sdk/identity/azpki/ci.yml

```diff
@@ -0,0 +1,28 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  branches:
+    include:
+    - main
+    - feature/*
+    - hotfix/*
+    - release/*
```

Comment:
Since I've retargeted the branch, can you please add an entry for `privatepreview/*` to this list.